script_library/deployment-actions-kvm.sh

```diff
@@ -30,6 +30,10 @@ function build_images(){
     echo "Nothing will happen if developer mode is not set"
     run_ansible ${socok8s_absolute_dir}/playbooks/generic-build_images.yml -e "build_osh_images=yes"
 }
+function setup_caasp_workers_for_openstack(){
+    echo "Ensuring caasp workers can be used for openstack"
+    run_ansible ${socok8s_absolute_dir}/playbooks/generic-setup_caasp_workers_for_openstack.yml
```

Comment:
This will fail the shellcheck travis test, needs to be quoted

Reply:
I can try having a go (pun intended as it's a bash script) at wrapping those.